src/main/java/time/TimeInCapitals/showtime/LocalTimeService.java

```diff
@@ -36,4 +55,9 @@ public String getTimeAndDate() {
 		return localTimeAndDate;
 	}
 
+	public boolean existsByKey(String key) {
+		
```

Comment:
Niepotrzebna pusta linia, to jednolinijkowa metoda.

Reply:
Poprawione